SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialDispatcher.cpp

```diff
@@ -24,6 +36,12 @@ void USpatialDispatcher::ProcessOps(Worker_OpList* OpList)
 	for (size_t i = 0; i < OpList->op_count; ++i)
 	{
 		Worker_Op* Op = &OpList->ops[i];
+
+		if (IsExternalSchemaOp(Op)) {
+			ProcessExternalSchemaOp(Op);
+			continue;
```

Comment:
@Vatyx @m-samiec comments on this approach to splitting out other pipeline (+ functions below) would be welcome

Reply:
I do like that this change touches way less of this function.